script/unit_test.sh

```diff
@@ -2,6 +2,7 @@
 set -ex
 coverage run --source=. -m unittest discover -s test/unit-test
 coverage report -m
+codecov -F unit
 codecov
```

Comment:
`codecov` can be removed. We have the required `codecov -F unit` in the previous line.